portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,115 @@
 <!DOCTYPE html>
 <html>
-  <head>
+<div style= "background-color: white;">
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
-  </head>
+  
+    <header style="position: sticky;top: 0;">
+       
+    </header>
+    <main style = "font-size: 30px;" >
+        My portfolio: By Dalen Witthoft
+    </main>
+     
+</div>
+
   <body>
+
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <img src = "/images/rimuru.png" style="width:5=100px;height:200px;">
+      <p>Click here to get a multilingual random message:</p>
+      <button onclick="addRandomGreeting()">Hello!</button>
       <div id="greeting-container"></div>
     </div>
+
+    <p style= "font-size: 20px">
+        Hi,
+        I am Dalen Witthoft and I am a rising sophmore at Boston University. I am majoring in 
+        Computer Science, but also have a passion for learning languages and about other cultures. I 
+        am currently studying Japanese and plan on expanding to Korean and Chinese before I graduate.
+        In my free time I love playing video games, watching anime, and watching K-Dramas. I also enjoy 
+        checking out what is going on in the competetive coding world as I did that throughout highschool. I
+        always find it interesting to see what strategies the top comptetive coders use to solve problems and 
+        see if I can adapt any of those approaches to my school work or personal projects. Thanks for checking out
+        my About Me website. :)
+    </p>
+
+    <div>
+        <h1>Experience  </h1>
+        <ul><img src= "/images/check.png" style="width:10px;height:20px;"> 4 Years of CS club in HS </ul>
+        <ul><img src= "/images/check.png" style="width:10px;height:20px;"> ID tech camp focusing on C#</ul>
+        <ul><img src= "/images/check.png" style="width:10px;height:20px;"> A&M engineering camp focusing on python</ul>
+        <ul><img src= "/images/check.png" style="width:10px;height:20px;"> Google's CSSI camp @ MTV: built a web app</ul>
+    </div>
+
+    <!-- Container for the image gallery -->
+<h1> Here are some cool pictures of the places I have been to around the World! </h1>
+
+<div class="container">
+
+  <!-- Full-width images with number text -->
+  <div class="mySlides">
+    <div class="numbertext">1 / 6</div>
+      <img src="/images/venice2.jpg" s
+  </div>
+
+  <div class="mySlides">
+    <div class="numbertext">2 / 6</div>
+      <img src="/images/empire3.jpg" s
+  </div>
+
+  <div class="mySlides">
+    <div class="numbertext">3 / 6</div>
+      <img src="/images/mtfuji2.jpg" s
+  </div>
+
+  <div class="mySlides">
+    <div class="numbertext">4 / 6</div>
```

Comment:
Perhaps consider a different way of doing this rather than hard-coding the slide index and the total number of slides. It may make it easier to add lots of pictures quickly in the future.

Reply:
Is there a way of doing that in html, I have worked with something called jinja that can maybe do   that if we have a backend setup using pyhton or java, but that was mainly to implement an API.